.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -1,5 +1,15 @@
 name: "Validate Gradle Wrapper"
-on: [ push, pull_request ]
+on:
+  push:
+    paths:
+      - 'gradlew'
+      - 'gradlew.bat'
+      - 'gradle/wrapper/'
+  pull_request:
+    paths:
+      - 'gradlew'
+      - 'gradlew.bat'
+      - 'gradle/wrapper/'
```

Comment:
If we can remove the check for `Validation`?

Reply:
This is probably correct, but the validation is also very quick so I don't think it's a huge savings either.

Reply:
Cause there might be some flaky failures from this action.